### ISSUE TITLE
Customizable block colors is not working

### ISSUE THREAD

**Issue by Zatan16** [ScratchAddons/ScratchAddons#6517]
### Description

The customizable block colors addon isn't working as intended. The colors are not matching the colors that were set in the settings.

### Steps to reproduce

1. Turn on the corresponding settings in Customizable Block Colors addon
2. Open [scratch project editor](http://mit.scratch.edu/create)

### Demonstration(s) and additional context

[Extension Screenshot](https://www.mediafire.com/view/44m7ndjdpy1hd9c/Cusomizable_Block_Colors_Screenshot.png/file)
[Block Palette Screenshot](https://www.mediafire.com/view/pza3ell6nex2k8d/Scratch_Block_Palette_Screenshot.png/file)

### Environment

Browser: Chrome 115.0.5790.170
Scratch Addons version: 1.33.3
Operating system: Ubuntu 22.04 LTS

**Comment by Zatan16** [comment]
![Screenshot from 2023-08-13 10-52-46](https://github.com/ScratchAddons/ScratchAddons/assets/79738479/bdac4db7-e2bc-4bcc-8e25-b2406e34d41e)

**Comment by DNin01** [comment]
It's possible that your browser has some sort of "auto dark mode" feature. Do you have "Editor dark mode and customizable colors" turned on? If the editor still looks dark when it's off, my assumption is correct.

**Comment by Zatan16** [comment]
Ill check my computer tomorrow and reply. As far as I remember, I think I have downloaded an extension for that.

**Comment by WorldLanguages** [comment]
Please disable all other browser extensions and any browser features such as "automatic dark mode" and try again

**Comment by mybearworld** [comment]
> Ill check my computer tomorrow and reply. As far as I remember, I think I have downloaded an extension for that.

Is it Dark Reader? That tends to do things like that. Even if not, most extensions that do this have a feature that allows you to disable this for certain pages.

**Comment by Zatan16** [comment]
Yes it is, ive just checked. I think I forgot to disable scratch in it.

**Comment by Zatan16** [comment]
I think the issue can be closed now.
